src/System.Linq.Expressions/src/System/Linq/Expressions/ExpressionStringBuilder.cs

```diff
@@ -354,9 +313,9 @@ protected internal override Expression VisitListInit(ListInitExpression node)
                 {
                     Out(", ");
                 }
-                Out(node.Initializers[i].ToString());
```

Comment:
This causes allocation of another `ExpressionStringBuilder` instance with a subsequent dispatch into `VisitElementInit` for no particular reason.